### PR TITLE
remove kind frontmatter - multiple pages

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do? What is the motivation?

the kind param is no longer a user-defined frontmatter. it is built-in to be one of `home`, `page`, `section` [etc.](https://gohugo.io/methods/page/kind/)

removes `kind` frontmatter from `continuous_testing/`, `data_jobs/`, `data_streams/`, `data_security/`, `dynamic_instrumentation/`, `metrics/`,  `monitors/`, `network_monitoring/`, `notebooks/`, `observability_pipelines/`, `opentelemety/`, `partners/`, `profiler/` pages

previews: (no changes expected):
- continuous_testing pages from external sources should still load.
   - e.g. [azure_devops_extension](https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-5/continuous_testing/cicd_integrations/azure_devops_extension), circleci_orbs, configurations, and [github_actions](https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-5/continuous_testing/cicd_integrations/github_actions) 
- https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-5/
- https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-5/ja
- https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-5/ko
- https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-5/fr

### Merge instructions

- [x] Please merge after CorpWeb review

### CODE REVIEW

File: content/en/security/threats/windows_expressions.md
Comment (Jackie-Robson) [inline content/en/security/threats/windows_expressions.md:1]: is this in scope of the PR? I'm not 100% on front matter still, just checking.

File: local/bin/py/build/actions/pull_and_push_file.py
Comment (Jackie-Robson) [inline local/bin/py/build/actions/pull_and_push_file.py:1]: same question with this.

File: content/en/security/threats/windows_expressions.md
Comment (StefonSimmons) [reply content/en/security/threats/windows_expressions.md:1]: i merged master into this branch. the changes are gone now

Comment (StefonSimmons) [reply content/en/security/threats/windows_expressions.md:1]: removed unintentional changes from this file. thanks for the thorough review!

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (StefonSimmons) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

File: local/bin/py/build/actions/pull_and_push_file.py
Comment (StefonSimmons) [reply local/bin/py/build/actions/pull_and_push_file.py:1]: removed the changes here as they are out of scope for this PR. Thanks!

**Review by devindford — APPROVED** [review]
LGTM, spot checked a few of the pages and everything is loading as expected

**Review by cswatt — APPROVED** [review]
